index.html

```diff
@@ -80,163 +81,176 @@ <h3 id="Background">Background</h3>
+						<li><strong>csv-validator-gui</strong> shell script on Unix/Linux.</li>
 					</ol>
 					You should then see the following:
 					</p>
-					<img src="images/1.3.0/main-screen.png" alt="The basic GUI, boxes to enter file names for data and schema, 'Validate' button, text box for ouput" id="GUI">
+					<img src="images/1.4.0/main-screen.png" alt="The basic GUI, boxes to enter file names for data and schema, 'Validate' button, text box for output" id="GUI">
 				</div>
 				<div>
 					<h3>Selecting metadata and schema for validation</h3>
 					<p>
-					To validate data you must supply a full filepath for the csv  file to be validated, and for the schema it is to be validated against. 
-					You can either type, or copy-and-paste, the filepath into the appropriate box, these are labelled "CSV file" and "CSV Schema file" respectively.
-					Alternatively, click the buttons labelled "..." (highlighted in the image below): 
+					To validate data, you must supply a full filepath of the csv file to be validated, and of the schema it is to be validated against.
+					There are multiple ways that you can supply the filepaths, you can either:
+					<ol>
+						<li>type, or copy-and-paste, the filepath into the appropriate box, these are labelled "CSV file" and "CSV Schema file" respectively.</li>
+						<li>select the files via a dialog window.</li>
+						<li>drag and drop the files onto the application.</li>
+					</ol>
+					<h4>Select the files via the dialog window</h4>
+					In order to select the files via a dialog window, click the buttons labelled "Choose..." (highlighted in the image below):
 					</p>
-					<img src="images/1.3.0/gui-file-selects-highlighted.png" alt="Part of GUI, showing in detail the boxes to enter file names for data and schema, file open dialogue buttons highlighted in yellow" id="GUI-detail1">
+					<img src="images/1.4.0/gui-file-selects-highlighted.png" alt="Part of GUI, showing in detail, the boxes to enter file names for data and schema, file open dialog buttons highlighted in yellow" id="GUI-detail1">
 					<p>
-					This will open up a standard File picker dialogue.  the dialog only shows the files of the type being chosen. You can navigate to the file you want to choose
+					This will open up a standard File picker dialog. The dialog only shows the files of the type being chosen. You can navigate to the file you want to choose
 					in the file system and select it:
 					</p>
-					<img src="images/1.3.0/file-open.png" alt="Standard file open dialogue, with CSV file selected" id="GUI-detail2">
-					<p>
-					Clicking "Save" will populate the related text box:
-					</p>
-					<img src="images/1.3.0/filepath-text-box-populated.png" alt="Part of GUI, showing completed CSV file text box" id="GUI-detai3l">
+					<img src="images/1.4.0/file-open.png" alt="Standard file open dialog, with CSV file selected" id="GUI-detail2">
```

Comment:
Are these images already in place? I had tended to capture the screenshots on Windows OS to be consistent. Do you want to do that as part of this PR? or separately?

Reply:
Ah, yes. Would you mind taking some new screenshots when we're close to release?

Cool. I'll create a 1.4.0 folder and put just the ones that need to be updated, in it

Reply:
Sure, maybe just raise a jira task about "capture new version screenshots" so we can track it